staging/src/k8s.io/cli-runtime/README.md

```diff
@@ -0,0 +1,29 @@
+# cli-runtime
+
+Set of helpers for creating kubectl plugins as well as kubectl compatible commands.
```

Comment:
Given that we are planning on dog-fooding this package ourselves in kubectl, maybe: `Set of helpers for creating kubectl commands, as well as kubectl plugins`?